.buildkite/pipeline.yml

```diff
@@ -1,13 +1,13 @@
-publish-android-artifacts-docker-container: &publish-android-artifacts-docker-container
-  plugins:
-    - docker#v3.8.0:
-        image: "public.ecr.aws/automattic/android-build-image:v1.0.0"
-        propagate-environment: true
-        environment:
-          # DO NOT MANUALLY SET THESE VALUES!
-          # They are passed from the Buildkite agent to the Docker container
-          - "AWS_ACCESS_KEY"
-          - "AWS_SECRET_KEY"
+x-common-params:
```

Comment:
nit and side-note: quick thought that we might want to bikeshed this arbitrary name to come up with a name we all agree with and ultimately try to use the same dummy name for all the YAMLs in all the repositories. `x-common-params`, `common-params`, `yaml-anchors`, `custom-yaml-anchor-definitions`, `reusable-nodes`… there are plenty of ideas and all are valid, so we should just agree on one and stick to it. Seems like Gio ended up using `common-params` in his other PRs, so maybe that'll become our de-facto naming for those?

Reply:
I've gotten this from your [suggestion here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3764#discussion_r678088237). 🤷 If we want to stick to something specific, let's have that discussion in a p2 with higher visibility. I'd prefer not to keep working on this PR until then.